packages/gatsby/src/schema/types/node-interface.js

```diff
@@ -0,0 +1,67 @@
+const getOrCreateNodeInterface = schemaComposer => {
+  // TODO: why is `mediaType` on Internal? Applies only to File!?
+  // `fieldOwners` is an object
+  const InternalTC = schemaComposer.getOrCreateTC(`Internal`, tc => {
+    tc.addFields({
+      content: `String`,
+      contentDigest: `String`,
+      description: `String`,
+      fieldOwners: [`String`],
+      ignoreType: `Boolean`,
+      mediaType: `String`,
+      owner: `String`,
+      type: `String!`,
+    })
+  })
+  InternalTC.getITC()
+
+  const NodeInterfaceTC = schemaComposer.getOrCreateIFTC(`Node`, tc => {
+    tc.setDescription(`Node Interface`)
+    tc.addFields({
+      id: `String!`,
```

Comment:
Why make the `id` field `String`, not `ID`? [It is ID now as well.](    debugger
)

Reply:
Link doesn't work. I had www fail when it was ID, there were some String variables that were passing in.

Reply:
Hmm that's weird